scripts/nlp_api_master.py

```diff
@@ -211,6 +339,77 @@
                         "keras_nlp.models.FNetClassifier.preprocessor",
                     ],
                 },
+                {
+                    "path": "f_net_masked_lm",
```

Comment:
Use `fnet_` (for better SEO)

Reply:
Actually this might break some stuff related to our preset table.

https://github.com/keras-team/keras-io/blob/22c9db7c71139fd128fd31213de7018d8b9fa1a2/scripts/render_nlp_tags.py#L32-L34

https://github.com/keras-team/keras-nlp/blob/7cbcbed0adbbf25af2324735c971591fc832f819/keras_nlp/models/f_net/f_net_presets.py#L25

We might want to leave this as is, unless we can figure out a way to deduce the "path" for model symbols totally in this repo, without relying on keras-nlp metadata. (Likely would be cleaner if we could find a way to do this)

Reply:
mark this as resolved.